### PR TITLE
add tests for our exceptions

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: tests/test_exceptions.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    captured = capsys.readouterr()
+    assert captured.out == '{}\n'.format(test_message)
+
+
+def test_docloadingerror(capsys):
+    test_message = 'this is DocLoadingError'
+    test_error = apypie.exceptions.DocLoadingError(test_message)
+    with pytest.raises(apypie.exceptions.DocLoadingError) as e:
+        raise test_error
+        assert e.message == test_message
```
Comment (ekohl) [inline tests/test_exceptions.py]: Shouldn't this be outside of the `with` context?

Comment (evgeni) [reply tests/test_exceptions.py]: probably, yeah

Comment (evgeni) [reply tests/test_exceptions.py]: well, no, `e` is not defined outside the `with`

Comment (ekohl) [reply tests/test_exceptions.py]: It is because Python doesn't clean its scope. Because of the raise this is unreachable code now.

Comment (evgeni) [reply tests/test_exceptions.py]: ah, me--

Comment (evgeni) [reply tests/test_exceptions.py]: updated!

**Comment by evgeni** [comment]
@sean797 now tests break, am I using your exception wrong?

**Comment by evgeni** [comment]
@sean797 this should pass tests now, am I using the exception as you intended to?

**Pull request merged**